src/config.js

```diff
@@ -0,0 +1,1603 @@
+/** @babel */
```

Comment:
We try not to use Babel anymore.  Looks like the only thing you have to change is how you're exporting the class.

Reply:
Fixed!